### ISSUE TITLE
Setting an empty collection as the items of SpDropListPresenter gives an error

### ISSUE THREAD

**Issue by jordanmontt** [pharo-spec/Spec#944]
When you call the method `emptyList` of a SpDropListPresenter or call `aSpDropDownList items: #( )` it gives the following error:
"SubscriptionOutOfBounds: 1".
Actually, the error occurs each time that you set an empty list as the items on an SpDropListPresenter.

**Comment by Whalehead99** [comment]
@jordanmontt please see issue #1145.  Does this happen when you _change_ dropList items to an emptyList... (i.e. from a previous list with items) or simply starting it as empty?  My issues seem to be SpDropListMorphic is holding on to a previous index # or the previous size of my items... and erroring when the items collection isn't 'long' enough to support selectIndex: anInteger (where anInteger is the size of the previous items collection).  When I go from 1 item to 0 items in the items collection, I also get the same error you are describing.  -Russ

**Comment by jordanmontt** [comment]
@Whalehead99 Yes, it also happens when you change the items of the drop list from a collection with items to an empty collection.

**Comment by jordanmontt** [comment]
Hello @estebanlm were you able to work on this one? We have some bugs on Moose related to this issue.

**Comment by estebanlm** [comment]
not yet
